app/celery/letters_pdf_tasks.py

```diff
@@ -167,6 +169,9 @@ def collate_letter_pdfs_to_be_sent():
                 queue=QueueNames.PROCESS_FTP,
                 compression='zlib'
             )
+        current_app.logger.info(f"finished collate-letter-pdfs-to-be-sent processing for postage class {postage}")
+
+    current_app.logger.info("finished collate-letter-pdfs-to-be-sent")
```

Comment:
I think we can already tell that a task has finished from this log line https://github.com/alphagov/notifications-api/blob/master/app/celery/celery.py#L36 but it also doesn't do any harm to have more logging